typescript/sdk/src/hook/HyperlaneHookDeployer.ts

```diff
@@ -81,8 +84,14 @@ export class HyperlaneHookDeployer extends HyperlaneDeployer<
       config.type === HookType.FALLBACK_ROUTING
     ) {
       hook = await this.deployRouting(chain, config, coreAddresses);
+    } else if (config.type === HookType.PAUSABLE) {
+      hook = await this.deployContract(chain, config.type, []);
+      await this.transferOwnershipOfContracts(chain, config.owner, { hook });
```

Comment:
Can you confirm that Pausable is the only ISM type for which we want to transfer ownership after deployment?

Reply:
its not the *only* one but we already do this in the `deployIgp` and `deployRouting` functions

Reply:
its actually a good point, it seems like it may be missing from a few places...